fla/layers/kda.py

```
@@ -273,3 +274,87 @@ def forward(
 
         return o, None, past_key_values
+
+
+class KimiDeltaAttentionWithCP(KimiDeltaAttention):
+    def __init__(
+        self,
+        hidden_size: int = 2048,
+        expand_v: float = 2,
+        head_dim: int = 256,
+        num_heads: int = 6,
+        num_v_heads: int = None,
```

Ruff (0.14.11)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)